WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -469,6 +472,33 @@ private extension OrderListViewController {
         childView.removeFromSuperview()
         childController.removeFromParent()
     }
+
+    /// Empty state config
+    ///
+    func createFilterConfig() ->  EmptyStateViewController.Config {
+        guard let filters = viewModel.filters, filters.numberOfActiveFilters != 0 else {
+            return emptyStateConfig
+        }
+
+        return noOrdersMatchFilterConfig()
+    }
+
+    /// Creates EmptyStateViewController.Config for no orders matching the filter empty view
+    ///
+    private func noOrdersMatchFilterConfig() ->  EmptyStateViewController.Config {
```

Comment:
nit: this function is already in a private extension, and has an extra space 🙂 
```suggestion
    func noOrdersMatchFilterConfig() -> EmptyStateViewController.Config {
```